docker/docker-compose.yml

```diff
@@ -40,4 +40,6 @@ services:
         condition: service_started    
     environment:
       - JSONRPC_HTTP_URL=http://zkevm-mock-l1-network:8545
-      - L2_CHAIN_ID=2955
\ No newline at end of file
+      - L2_CHAIN_ID=2955
+      - DEPLOYER_PRIVATE_KEY="0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
+      - DEPLOYER_ADDRESS="0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266"      
```

Comment:
discussed offline. we don't need the address. priv key can be used to derive the address